Makefile

```diff
@@ -57,11 +66,6 @@ generate-testdata: ## Update/generate the testdata in $GOPATH/src/sigs.k8s.io/ku
 generate-vendor: ## (Deprecated) Update/generate the vendor by using the path $GOPATH/src/sigs.k8s.io/kubebuilder-test
 	GO111MODULE=off ./generate_vendor.sh
 
-.PHONY: generate-setup
-generate-setup: ## Current workarround to generate the testdata with the correct controller-gen version
-	- rm -rf $(CONTROLLER_GEN_BIN_PATH)
-	- GO111MODULE=on go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.5
```

Comment:
You don't need to remove this section. I imagine that what you can do here is
1) create a temp directory (e.g. using https://golang.org/pkg/io/ioutil/#TempDir)
2) cd to that directory
3) run go get with go modules on
4) cd to the previous working directory
5) remove the temp dir.

Reply:
The idea of this makefile target was just to ensure that the correct version will be used.
Note that we we just need to remove any pre-existent installation to ensure that the version defined in the makefile will be used. So, the point is that it in POV it is not required at all and because of this I keep just the rm -rf line. See: https://github.com/kubernetes-sigs/kubebuilder/pull/1454/files#diff-b67911656ef5d18c4ae36cb6741b7965R57